### DISCUSSION TITLE
How do I using av.open to open a ByteIO object?

### DISCUSSION THREAD

**Discussion by fake-warrior8** [PyAV-Org/PyAV#999] [1. Help]
**IMPORTANT:** Be sure to replace all template sections {{ like this }} or your issue may be discarded.

## Overview

I want to use av.open to open a ByteIO object. However, video_byte=av.open(buff) (where buff=ByteIO(some_bytes)) will output some errors. For example, video_byte.streams.video[0].time_base is None while video_mp4=av.open(some_mp4), video_mp4.streams.video[0].time_base will give a right number.

## Expected behavior

video_byte = av.open(buff) (where buff=ByteIO(some_bytes))  and video_mp4=av.open(some_mp4) will work the same, which means their attributes are the same.

## Actual behavior

video_byte.streams.video[0].time_base is None

Traceback:
```
No exceptions.
```

## Investigation

```
buff=ByteIO(some_bytes)
buff.seek(0) # found in Google
video_byte = av.open(buff) 
```

## Research

I have done the following:

- [* ] Checked the [PyAV documentation](https://pyav.org/docs)
- [*] Searched on [Google](https://www.google.com/search?q=pyav+how+do+I+foo)
- [*] Searched on [Stack Overflow](https://stackoverflow.com/search?q=pyav)
- [* ] Looked through [old GitHub issues](https://github.com/PyAV-Org/PyAV/issues?&q=is%3Aissue)
- [ ] Asked on [PyAV Gitter](https://gitter.im/PyAV-Org)
- [ ] ... and waited 72 hours for a response.

## Additional context

{{ Add any other context about the problem here. }}

**Comment by Minipada** [comment]
Any solution?

**Reply by jberclaz** [reply]
I think you have to specify the video format, i.e. `av.open(buff, format="mp4")`.
